README.md

```diff
@@ -104,6 +104,29 @@ docker run --rm -it --env GITHUB_TOKEN=<TOKEN> rancher/ecm-distro-tools tag_rke2
 
 See seperate [documentation](./docs/test_pad.md) for test-pad tool.
 
+## GitHub action
+
+This repository provides the "Setup ecm-distro-tools" GitHub action. 
+It downloads the assets belonging to the specified release to a temporary directory,
+and adds the directory to the `PATH`.
+
+### Usage
+
+The action can be run on ubuntu-latest runners. 
+The `version` parameter is required.
+
```

Comment:
I think you should mention that the token is recommended.

Reply:
Added a line mentioning this. Thanks.